packages/tokens/src/themeToTokensObject.ts

```diff
@@ -12,7 +12,7 @@ export function themeToTokensObject<TTheme extends Theme>(theme: TTheme): Record
   const tokens = {} as Record<keyof TTheme, string>;
   const keys = Object.keys(theme) as (keyof TTheme)[];
   for (const key of keys) {
-    tokens[key] = `var(--${key})`;
+    tokens[key] = `var(--${String(key)})`;
```

Comment:
This was breaking on type-check:
![image](https://github.com/microsoft/fluentui/assets/8649804/327121d0-50a8-461f-ab91-e53ab662b86b)